README.md

```diff
@@ -26,18 +40,140 @@ A slackbot which reads an excel file from sharepoint and returns a specific colu
 4. Add Bot Token Scopes 
     - chat:write
     - commands
+5. go to Slash commands and press Create New Command
+    - Command name: whatever you want it to be
+    - Fill in the required fields and press Save.
+
 
 ### Code 
-1. Clone 
-2. npm i in the project directory 
-3. create a .env file 
+1. Make a folder or move into a prefered one
+```yaml
+mkdir <your-folder-name>
+cd <your-folder-name>
+```
+2. Install needed dependencies and start the project
+```yaml
+npm install -g typescript
+npm init -y
+npm install typescript --save-dev
+npm install @slack/bolt
+npm install dotenv --save
+npx tsc --init
+```
+in your package.json file, dependencies should be @slack/bolt and devDependencies should be typescript and dotenv
 
-***.env file***
+2. Make a new folder called src and a file inside it called app.ts
+3. go into your package.json, delete the exsisting "test" script and make a new one for example:
+```yaml
+"scripts": {
+    "start": "tsc && node build/app.js"
+}
+```
+4. create a .env file
 ```yaml
 SLACK_BOT_TOKEN = <Your Bot User OAuth token>
 SLACK_APP_TOKEN = <Your App-Level Token>
 ```
 
 ### Start
-1. Go to your project folder and type npm start
-2. CTRL + C to stop the bot 
+1. Make sure you have downloaded all the needed dependencies 
+2. Make sure you're in the project root
+```yaml
+cd /<your-folder-name>/slackbot-sharepoint-excel-reader
+```
+3. type npm start, it should create a folder "build" and a file inside of it called "app.js" and give you this output in your terminal
+```yaml
+npm start
+
+> slackbot0.0.1 start
+> tsc && node build/app.js
+
+⚡️ Bolt app is running in Socket Mode!
+```
+
+Go into slack and try typing /"your-command" in any channel. The bot should respond "Hello + your name 👋"
+You can stop the bot by pressing CTRL + C in your terminal. After that, the bot should stop responding to your command
+
+this is what your project structure should look like:
```

Comment:
Do we need this description of project structure, you get the whole project structure when you clone it from github